### PR TITLE
Test automation dev guidlines

### PR DESCRIPTION
**Pull request by gurevichdmitry**

### CODE REVIEW

**Review by DaveSys911 — APPROVED** [review]
LGTM

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+In the github, fork [cloudbeat repo](https://github.com/elastic/cloudbeat). After forking the repo, clone and install dependencies.
+
+```shell
+git clone https://github.com/<yourname>/cloudbeat.git
+cd cloudbeat/tests
+poetry install
+```
+### Configuring IDE
+
+Tests development may be done using any IDE like [visual studio code](https://code.visualstudio.com/) or [pycharm](https://www.jetbrains.com/pycharm/promo/?source=google&medium=cpc&campaign=14124132918&gclid=CjwKCAjw9LSSBhBsEiwAKtf0n1GGB55o_rxfYFVmBzRYUqbz_g5YhZdP8Th1iKXFsaRl43rEz_h7ehoCl2wQAvD_BwE).
```
Comment (kfirpeled) [inline tests/README.md:98]: nit remove parameters from link. looks too suspicious 
https://www.jetbrains.com/pycharm/

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
+# Cloudbeat Automated Tests Framework (C-ATF)
+> 
+
+This project provides an ability to develop component and integration tests for Cloudbeat.
+
+
+## Getting started
+
+All installations below (except of docker desktop) are available via [Homebrew](https://brew.sh/).
```
Comment (kfirpeled) [inline tests/README.md:9]: missing installation for python

File: tests/README.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+If you have a different OS, in order to install, please go forward through instructions provided in links. 
+
+1. Install [Docker Desktop](https://docs.docker.com/desktop/mac/install/)
+
+
+2. Install [JUST](https://github.com/casey/just) command runner
+
+```shell
+brew install just
```
Comment (kfirpeled) [inline tests/README.md:18]: pay attention that `brew` is available only on mac machines
this guideline should serve all user's platforms.

File: tests/README.md (hunk 2 above)
Comment (gurevichdmitry) [reply tests/README.md:9]: Poetry package manager installs python if not installed yet.

File: tests/README.md (hunk 1 above)
Comment (gurevichdmitry) [reply tests/README.md:98]: The link will be updated in the next PR.

File: tests/README.md (hunk 3 above)
Comment (gurevichdmitry) [reply tests/README.md:18]: As it is written in Getting started section, users that are no working on mac machines shall go forward through provided links and find relevant installation command / package for their os platform.

**Pull request merged**